modules/openapi-generator/src/test/java/org/openapitools/codegen/python/PythonClientTest.java

```diff
@@ -494,7 +494,7 @@ public void testRecursiveExampleValueWithCycle() throws Exception {
         final OpenAPI openAPI = TestUtils.parseFlattenSpec("src/test/resources/3_0/issue_7532.yaml");
         final PythonExperimentalClientCodegen codegen = new PythonExperimentalClientCodegen();
         codegen.setOpenAPI(openAPI);
-        Schema tree = openAPI.getComponents().getSchemas().get("Tree");
+        Schema tree = openAPI.getComponents().getSchemas().get("Forest");
```

Comment:
How about a comment mentioning that this node has a cycle.
Or naming it something like schemaWithACycleInAllOf?